nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -240,7 +240,7 @@ public boolean equals(Object o) {
     if (this == o) {
       return true;
     }
-    if (o == null || getClass() != o.getClass()) {
+    if (!(o instanceof AccessPath)) {
```

Comment:
Fwiw, you could also mark the class as `final`.

Reply:
I like the `instanceof` check better anyway, but I'll also make the class `final`.